spec/models/entry_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe Entry do
+##########################################################################################
+## Factorygirl generate
+##########################################################################################
+  let (:e) {create(:entry)}
+
+##########################################################################################
+## Elo validity
+##########################################################################################
+  it "has a valid elo of type Integer" do
+    e.elo.should be_an(Integer)
+  end
+
+##########################################################################################
+## Association with league cases
+##########################################################################################
+  it "is valid if it belongs to a league" do
+    e.valid?
+    expect(e).to be_valid
```

Comment:
seems like you're just testing the factory is valid here, if that's the intent probably just call this test: it "has a valid factory"